lib/edgehog_device_forwarder/application.ex

```diff
@@ -11,7 +11,8 @@ defmodule EdgehogDeviceForwarder.Application do
     children = [
       EdgehogDeviceForwarderWeb.Telemetry,
       {Phoenix.PubSub, name: EdgehogDeviceForwarder.PubSub},
-      EdgehogDeviceForwarderWeb.Endpoint
+      EdgehogDeviceForwarderWeb.Endpoint,
+      EdgehogDeviceForwarder.Connections
```

Comment:
I'd assume that the Endpoint uses the Connections (since it will call into that to register stuff), so it would be better to start Connections before the Endpoint